config/crd/bases/psmdb.percona.com_perconaservermongodbs.yaml

```diff
@@ -6304,6 +6312,21 @@ spec:
                           format: int32
                           type: integer
                       type: object
+                    replsetOverrides:
```

Comment:
Please add it as an example into cr.yaml

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1640/commits/19b3be325878d604f018286a61743fa32c8b58d9